tests/tasks/cassettes/test_transaction_not_in_cuenca.yaml

```diff
@@ -0,0 +1,174 @@
+interactions:
+- request:
+    body: '{"id": "5ea709e72465c3978629618e", "created_at": "2020-04-27T16:35:51.332667",
+      "updated_at": "2020-04-27T16:35:51.334089", "stp_id": null, "fecha_operacion":
+      "2020-04-24", "institucion_ordenante": "40021", "institucion_beneficiaria":
+      "90646", "clave_rastreo": "HSBC000089", "monto": 100000, "nombre_ordenante":
+      "CARLOS JAIR  AGUILAR PEREZ", "tipo_cuenta_ordenante": 40, "cuenta_ordenante":
+      "021650040600992156", "rfc_curp_ordenante": "AUPC890116DU0", "nombre_beneficiario":
+      "CARLOS JAIR CTA CUENCA", "tipo_cuenta_beneficiario": 40, "cuenta_beneficiario":
+      "646180157026913146", "rfc_curp_beneficiario": "", "concepto_pago": "Andy",
+      "referencia_numerica": 230420, "empresa": "TAMIZI", "estado": "Estado.created",
+      "version": null, "speid_id": "SR1dOSXPfozmqpLbleWPmHcM", "folio_origen": null,
+      "tipo_pago": null, "email_beneficiario": null, "tipo_cuenta_beneficiario2":
+      null, "nombre_beneficiario2": null, "cuenta_beneficiario2": null, "rfc_curpBeneficiario2":
+      null, "concepto_pago2": null, "clave_cat_usuario1": null, "clave_cat_usuario2":
+      null, "clave_pago": null, "referencia_cobranza": null, "tipo_operacion": null,
+      "topologia": null, "usuario": null, "medio_entrega": null, "prioridad": null,
+      "iva": null, "compound_key": "HSBC000089:20200424", "events": ["Event object"]}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Authorization:
+      - Basic QUszNkVBUVhJbHVyZzUzcFJqY3Z0eDMwOkVhLTZKbXBuUlgzWktpNG9HSHBzbXJvR0VFUFUxdmd1MkVxMzYyQjI3S2lKRzJhTEdrdUdoM3BZZkc3MjV5RFNOeFFuMEw5MnpDb0Z2RWRucnIxWWJ3
```

Comment:
You can add this to avoid adding your Authorization in any file:
https://github.com/cuenca-mx/cuenca/blob/260ed34868d69c89bf0104e2b1a13996c94abb49/tests/conftest.py#L51-L58